mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/server/master/persistence/MantisStorageProviderAdapter.java

```diff
@@ -50,6 +66,27 @@
 public class MantisStorageProviderAdapter implements IMantisStorageProvider {
 
     private static final Logger logger = LoggerFactory.getLogger(MantisStorageProviderAdapter.class);
+    private static final ObjectMapper mapper = new ObjectMapper();
+    private static final String JOB_STAGEDATA_NS = "MantisJobStageData";
+    private static final String ARCHIVED_JOB_STAGEDATA_NS = "MantisArchivedJobStageData";
+    private static final String WORKERS_NS = "MantisWorkers";
+    private static final String ARCHIVED_WORKERS_NS = "MantisArchivedWorkers";
+    private static final String NAMED_JOBS_NS = "MantisNamedJobs";
+    private static final String NAMED_COMPLETEDJOBS_NS = "MantisNamedJobCompletedJobs";
+    private static final String ACTIVE_ASGS_NS = "MantisActiveASGs";
+    private static final String TASK_EXECUTOR_REGISTRATION = "TaskExecutorRegistration";
+    private static final String DISABLE_TASK_EXECUTOR_REQUESTS = "MantisDisableTaskExecutorRequests";
+    private static final String CONTROLPLANE_NS = "mantis_controlplane";
+    private static final String NAMED_JOB_SECONDARY_KEY = "jobNameInfo";
+    private static final int WORKER_BATCH_SIZE = 1000;
+    private static final int WORKER_MAX_INDEX = 30000;
+
+    static {
+        mapper
```

Comment:
nit: use JsonSerializer from mantis-common-serde.

Reply:
The serializer here depends on `DateTimeModule` in jackson. I'd avoid adding it to JsonSerializer, actually eliminate from here but would like to understand better on why it's needed.